src/main/scala/cromwell/util/google/GoogleScopes.scala

```diff
@@ -1,12 +1,17 @@
 package cromwell.util.google
 
+import com.google.api.services.genomics.GenomicsScopes
 import com.google.api.services.storage.StorageScopes
+import scala.collection.JavaConverters._
 
 object GoogleScopes {
-  val Scopes = Vector(
-    StorageScopes.DEVSTORAGE_FULL_CONTROL,
-    StorageScopes.DEVSTORAGE_READ_WRITE,
-    "https://www.googleapis.com/auth/genomics",
-    "https://www.googleapis.com/auth/compute"
-  )
+  val Scopes = {
+    val scopes = Vector(
+      StorageScopes.DEVSTORAGE_FULL_CONTROL,
+      StorageScopes.DEVSTORAGE_READ_WRITE,
+      "https://www.googleapis.com/auth/genomics",
+      "https://www.googleapis.com/auth/compute"
+    ) ++ GenomicsScopes.all().asScala
+    scopes.sorted.distinct
```

Comment:
Does the order of the scopes not matter?  It looks like this will squash out duplicates but leave the `Vector` in an alphabetical but semantically arbitrary order.